src/main/docs/guide/views/templates/jstachio.adoc

```diff
@@ -0,0 +1,43 @@
+Micronaut Views JStachio includes api:views.jstachio.JStachioViewsRenderer[] which uses
+https://github.com/jstachio/jstachio[JStachio] A type-safe compiled Mustache based template engine.
+
+Add the `micronaut-views-jstachio` dependency to your classpath.
+
+dependency:micronaut-views-jstachio[groupId="io.micronaut.views"]
+
+The example shown in the <<views, Views>> section, could be rendered with the following JStachio 
+annotated model and template:
+
+[source,java]
+.src/main/java/myapp/HomeModel.java
+----
+//Normally you would put the JStacheConfig annotation on package-info or module-info
+//but it can be placed on the model class directly
+@JStacheConfig(using = MicronautJStacheConfig.class) 
+@JStache(path = "home")
+public record HomeModel(@Nullable String username, boolean loggedIn) {
+
+}
+----
+
+[source,html]
+.src/main/resources/views/home.mustache
+----
+include::{testsviewsJstachio}/resources/views/home.mustache[]
```

Comment:
Please, define `testsviewsJstachio` in gradle.properties You can generate the docs with `./gradlew docs`